client/login/magic-login/request-login-email-form.jsx

```diff
@@ -183,8 +183,8 @@ const mapState = ( state ) => {
 		showCheckYourEmail: getMagicLoginCurrentView( state ) === CHECK_YOUR_EMAIL_PAGE,
 		emailRequested: getMagicLoginRequestedEmailSuccessfully( state ),
 		userEmail:
-			getInitialQueryArguments( state ).email_address ||
-			getCurrentQueryArguments( state ).email_address,
+			getCurrentQueryArguments( state ).email_address ||
```

Comment:
I'm not sure why it was the other way around. It seems to me that `getCurrentQueryArguments` would never have been called.